cmd/root.go

```diff
@@ -65,7 +65,7 @@ func setFlags(cmd *cobra.Command, v *viper.Viper) {
 	)
 	flags.Int("cursorpos", 0, "Optional cursor position in the current file.")
 	flags.Bool("disable-offline", false, "Disables offline time logging instead of queuing logged time.")
-	flags.Bool("disableoffline", false, "(deprecated) Disables offline time logging instead of queuing logged time.")
+	flags.Bool("disableoffline", false, "")
```

Comment:
The info will be suppressed and replaced by `please use --disable-offline instead.`, as defined via `flags.MarkDeprecated()`, right?

Then we should not change the old info. As we still have the logic to handle these parameters, it is good to keep it for documentation purposes. Also the `(deprecated)` info is a good indicator, to spot deprecated params within the list.

Reply:
Because deprecated flags are hidden the old info might be supressed (removed) as well, it becomes useless since it won't be printed anywhere. The text `please use ....` will be in place when those flags are used.

Reply:
That's what I mean. It won't be visible to the user, but helps us as the maintainers, keep track of what these params are doing.

Reply:
got it